pkg/services/contexthandler/contexthandler.go

```diff
@@ -172,6 +172,11 @@ func (h *ContextHandler) Middleware(next http.Handler) http.Handler {
 			}
 		}
 
+		// this can be used by proxies to identify certain users
+		if h.features.IsEnabled(featuremgmt.FlagReturnUnameHeader) {
+			w.Header().Add("grafana-uname", reqContext.Login)
```

Comment:
FYI note this header will be rendered as `Grafana-Uname` due to canonical form enforced by Go.